tests/acceptance/features/bootstrap/WebUIUsersContext.php

```diff
@@ -253,6 +253,56 @@ public function theAdminResendsInvitationEmailUsingTheWebUI(
 		$this->usersPage->resendInvitationEmail($username, $this->getSession());
 	}
 
+	/**
+	 * @Then /^the (?:subadmin|administrator) should (not|)\s?be able\s? to see password field in new user form on the webUI$/
+	 *
+	 * @param string $shouldOrNot
+	 *
+	 * @return void
+	 */
+	public function subadminShouldBeAbleToSeePasswordFieldInNewUserForm($shouldOrNot) {
+		$expected = true;
+		$expectedMsg = "visible";
+		if ($shouldOrNot == "not") {
+			$expected = false;
+			$expectedMsg = "invisible";
+		}
+
+		$visible = $this->usersPage->isPasswordFieldOfNewUserVisible();
```

Comment:
I think $visible is a bit misleading.

```suggestion
		$isVisible = $this->usersPage->isPasswordFieldOfNewUserVisible();
```